.github/workflows/linux.yml

```diff
@@ -97,6 +102,12 @@ jobs:
         export OMPI_MCA_btl="self,vader"
         ./update_parameters.sh $GITHUB_WORKSPACE/build/aspect
 
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        path-to-lcov: ${{github.workspace}}/build/coverage_filtered.info
```

Comment:
I think all changes need to go into the `tests` workflow, because only in that one do we actually run the tests, the `indent+documentation` only compiles to update the parameters documentation.